sync2jira/downstream_issue.py

```diff
@@ -979,6 +979,14 @@ def _update_github_project_fields(client, existing, issue,
     for name, values in github_project_fields.items():
         fieldvalue = getattr(issue, name)
         if name == 'storypoints':
+            if not isinstance(fieldvalue, int):
```

Comment:
You are technically correct -- if `fieldvalue` is an `int`, then the "conversion" will proceed without error, and we'll end up in the same place as we do if we skip the `try` block -- however, my hope is that, once the upstream/intermediate-issue code is corrected, we will remove the `try` block and simply `continue` if `fieldvalue` is not an `int` (because it will be `None` in that case or otherwise _supposed_ to be excluded), so we'll need this check back at that point, so we might as well include it now.